agent/api/task_test.go

```diff
@@ -96,6 +96,28 @@ func TestDockerHostConfigVolumesFrom(t *testing.T) {
 	}
 }
 
+func TestDockerConfigLabels(t *testing.T) {
+	testTask := &Task{
+		Arn: "arn:aws:ecs:us-east-1:012345678910:task/c09f0188-7f87-4b0f-bfc3-16296622b6fe",
+		Containers: []*Container{
+			&Container{
+				Name: "c1",
+			},
+		},
+	}
+
+	config, err := testTask.DockerConfig(testTask.Containers[0])
+	if err != nil {
+		t.Fatal(err)
+	}
+	if !reflect.DeepEqual(config.Labels, map[string]string{
```

Comment:
Might want to move the inline map you have out to a named variable just for readability.